### DISCUSSION TITLE
Jest fails to import paste icons

### DISCUSSION THREAD

**Discussion by macdonmc** [twilio-labs/paste#2343] [Q&A] [answered]
I tried the recommended usage of `transformIgnorePatterns` in the paste docs which didn't work, so I looked at [this post](https://github.com/twilio-labs/paste/discussions/1589) which also did not fix the issue. Here's the error I'm getting in jest. BTW in my package json i do have `"type": "module"`

Any advice?

<img width="1052" alt="Screen Shot 2022-04-06 at 10 25 47 AM" src="https://user-images.githubusercontent.com/46056299/162032878-554c176e-f7e6-4e83-9d82-f441681356d6.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @macdonmc,

This is a troublesome issue wherein Jest understands ES6 modules, but doesn't expect them from compiled dependencies. Which part of [this post](https://github.com/twilio-labs/paste/discussions/1589#discussioncomment-911169) did you try and what was the outcome?

An alternative may be to change your icon imports to be from `/cjs/` instead of `/esm/` if all else fails. Hope this helps!

**Reply by macdonmc** [reply]
I tried every strategy in that post and none of them seemed to work, but importing from `/cjs/` did, thanks for that rec
